guides/release/routing/specifying-a-routes-model.md

```diff
@@ -9,118 +11,128 @@ Router.map(function() {
 });
 ```
 
-To load a model for the `favorite-posts` route, you would use the [`model()`](https://www.emberjs.com/api/ember/release/classes/Route/methods/model?anchor=model)
-hook in the `favorite-posts` route handler:
+When a user first visits the `/favorite-posts` route, the `model` hook in `app/routes/favorite-posts.js` will automatically run.
+In Ember, functions that automatically run during rendering or setup are commonly referred to as "hooks". 
+Here's an example of a model hook in use within a route:
 
 ```javascript {data-filename=app/routes/favorite-posts.js}
 import Route from '@ember/routing/route';
 
 export default Route.extend({
   model() {
-    return this.store.query('post', { favorite: true });
+    console.log('The model hook just ran!')
+    // write code to return a Promise or some static data
   }
 });
 ```
 
-Typically, the `model` [hook](../../getting-started/core-concepts/#toc_hooks) should return an [Ember Data](../../models/) record,
-but it can also return any [promise](https://www.promisejs.org/) object (Ember Data records are promises/),
-or a plain JavaScript object or array.
-Ember will wait until the data finishes loading (until the promise is resolved/) before rendering the template.
+`model` hooks have some special powers:
 
-The route will then set the return value from the `model` hook as the `model` property of the controller.
-You will then be able to access the controller's `model` property in your template:
+1. When you return data from this model, it becomes automatically available in the route's `.hbs` file as `this.model`
+2. A `model` hook can return either a JavaScript [Promise](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Using_promises), or static sample data, like a string, object, or array
+3. If you return a Promise from the model hook, your route will wait for the Promise to resolve before it renders the template
+4. `model` hooks work great with
+[Ember Data](../../models/),
+[`ember-ajax`](https://github.com/ember-cli/ember-ajax),
+or a JavaScript data fetching library of your choice
```

Comment:
In the spirit of keeping things as simple as possible, I think we may be able to eliminate this point as well.

If there's a feeling that we should point that this works well with data fetching tools, we may want to consider reaching for the simplest and most universal (to those unfamiliar with Ember) examples that will do the job 
(i.e., [fetch](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API), or potentially AJAX)

Talking about core browser APIs instead of these libraries is a big help at reducing the feeling that we're introducing too much complexity too early

Reply:
Agreed, revised